### PR TITLE
Fix permissions in debian package

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #3009

### CODE REVIEW

File: gulpfile.js
Comment (howels) [inline gulpfile.js:740]: This `chmod -R +xr` will make all files in the directory executable, whereas we only want the two binaries, `/opt/betaflight/betaflight-configurator/chrome_crashpad_handler` and `/opt/betaflight/betaflight-configurator/betaflight-configurator` to be executable - not bitmaps, libraries etc.
So the three commands are needed, then the capital X in `chmod -R +Xr /opt/betaflight/betaflight-configurator/` sets the *directory only* to be executable which means that the user can read which files are inside the directory.

**Review by howels — CHANGES_REQUESTED** [review]

**Review by howels — CHANGES_REQUESTED** [review]
After reading the rest of gulpfile.js I think we should use $`{LINUX_INSTALL_DIR}/${metadata.name}` in place of `/opt/betaflight/betaflight-configurator` because this path can change depending on OS preferences. Sorry to ask again.

**Review by howels — APPROVED** [review]
Tested and working perfectly now, appreciate the fix.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]
LGTM. Strange that this permission problem does not happen on all systems. For a future, if this continue growing, maybe is better to move the scripts to a external file. In this way we can do more complicated things.

**Comment by nerdCopter** [comment]
> [Betaflight-Configurator-Linux](https://github.com/betaflight/betaflight-configurator/suites/8517889752/artifacts/379554352) 

`10.9.0-debug-3d88148`: 

* Ubuntu 20.04.5 installation and execution succeeds.
* `.zst` issue for Debian remains.

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
